console/frontend/src/main/frontend/src/app/views/storage/storage-list/storage-list-dt/storage-list-dt.component.ts

```diff
@@ -15,5 +16,22 @@ import { LaddaModule } from 'angular2-ladda';
 export class StorageListDtComponent {
   @Input({ required: true }) message!: PartialMessage;
 
-  constructor(public storageService: StorageService) {}
+  protected storageService: StorageService = inject(StorageService);
+  private sweetAlert: SweetalertService = inject(SweetalertService);
+
+  moveMessage(): void {
+    this.sweetAlert
+      .Warning({
+        title: 'Move Message State',
+        text: 'Are you sure you want to move this message to Error?',
```

Comment:
```suggestion
        text: 'The message might still be processing in the background. Are you sure you want to move it to Error?',
```

I think this informs the user a bit more about having unwanted side-effects.